### PR TITLE
 DOC: add to documentation of orthogonal polynomials

### PR DESCRIPTION
**Pull request by person142**
Closes gh-6395.

### CODE REVIEW

File: scipy/special/orthogonal.py [hunk 1]
```diff
@@ -1041,8 +1193,34 @@ def he_roots(n, mu=False):
...
+
+    Returns
+    -------
+    He : orthopoly1d
+        Hermite polynomial.
+
+    Notes
+    -----
+    The polynomials `He_n` are orthogonal over :math:`(-\infty,
+    \infty)` with weight function :math:`e^{-(x/2)^2}`.
```
Comment (ev-br) [inline scipy/special/orthogonal.py]: Is the weight function correct as written -- is it (x/2)**2 or x**2/2 ? Ditto in the docstring of he_roots.

**Review by ev-br — COMMENTED** [review]
LGTM modulo a possible typo.

File: scipy/special/orthogonal.py (hunk 1 above)
Comment (person142) [inline scipy/special/orthogonal.py]: That's a typo; good catch.

**Comment by ev-br** [comment]
Thanks Josh

**Comment by person142** [comment]
Thank you Evgeni for taking the time to review a rather dry PR.

**Pull request merged**